packages/assets-controllers/src/index.ts

```diff
@@ -166,3 +166,14 @@ export type {
   MultichainBalancesControllerEvents,
   MultichainBalancesControllerMessenger,
 } from './MultichainBalancesController';
+
+export { MultichainAssetsController } from './MultichainAssetsController';
```

Comment:
Do we need to also add the `getDefaultMultichainAssetsControllerState` function to the list of exports?

``` suggestion
export {
  MultichainAssetsController,
  getDefaultMultichainAssetsControllerState
} from './MultichainAssetsController';
```